client/gutenberg/extensions/contact-form/index.js

```diff
@@ -42,18 +43,43 @@ export const settings = {
 			type: 'string',
 			default: '',
 		},
-		submit_button_text: {
+		submitButtonText: {
 			type: 'string',
 			default: __( 'Submit' ),
 		},
-		has_form_settings_set: {
+		backgroundButtonColor: { type: 'string' },
+		textButtonColor: { type: 'string' },
+		hasFormSettingsSet: {
 			type: 'string',
 			default: null,
 		},
 	},
 
 	edit: JetpackContactForm,
 	save: InnerBlocks.Content,
+	deprecated: [
+		{
+			attributes: {
+				submit_button_text: {
+					type: 'string',
+					default: __( 'Submit' ),
+				},
+				has_form_settings_set: {
+					type: 'string',
+					default: null,
+				},
+			},
+
+			migrate: function( attributes ) {
+				return {
+					submitButtonText: attributes.submit_button_text,
+					hasFormSettingsSet: attributes.has_form_settings_set,
+				};
+			},
```

Comment:
Toootally a nit but I'd compact a bit if it doesn't affect readability:

```js
migrate: ( { submit_button_text, has_form_settings_set } ) => {
	submitButtonText: submit_button_text,
	hasFormSettingsSet: has_form_settings_set,
},
```

🙃

Reply:
Updated, thanks.